src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementForm.tsx

```diff
@@ -50,7 +51,17 @@ const PermissionManagementForm = ({
   colonyDomains,
   onDomainSelected,
   userHasPermission,
+  currentUserRolesInRoot,
 }: Props) => {
+  const canSetPermissionsInRoot =
+    domainId === ROOT_DOMAIN_ID &&
+    currentUserRoles.includes(ColonyRole.Root) &&
+    (!userDirectRoles.includes(ColonyRole.Root) || rootAccounts.length > 1);
```

Comment:
Shouldn't this be `>= 1` ?

Reply:
Im actually not sure about this part of logic 🤔 This logic exist before, but now im not sure if its correct honestly...\
 1. Should we block possibility of  removing root permission by person which also holding root? 🤔  (probably question to product)
 2. Should we block possibility of removing root from last root? (also probably question to product)